### PR TITLE
pygraphviz: init at 1.2

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Compiling it takes years on my master, as it compiles graphviz and deps (down to systemd as far as I can see) for me.

Will compile as soon as I'm on my workstation again, if you guys want me to. Otherwise I will try to compile based on unstable.

### CODE REVIEW

**Comment by FRidh** [comment]
@matthiasbeyer when writing expressions like this I can recommend working on top of a checkout of unstable (check the nixpkgs-channels repo) or on top of a checkout of a (mostly finished) Hydra build, so you can use the cache for (most) dependencies.

**Comment by matthiasbeyer** [comment]
@FRidh Yep, I'll normally do this, though I was to lazy to rebase in this case...

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -10808,6 +10808,25 @@ in modules // {
...
+      url = "https://github.com/pygraphviz/pygraphviz/archive/${name}.tar.gz";
+      sha256 = "0jyd049wj994y1h9k3xwvjk9jwzi6vpdj10767kq8alz86hp4ak5";
+    };
+
+    propagatedBuildInputs = [ pkgs.graphviz pkgs.pkgconfig ];
+
+    meta = {
+      description = "Python interface to Graphviz graph drawing package";
+      homepage = https://github.com/pygraphviz/pygraphviz;
+      license = licenses.bsd;
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: this is invalid license, `bsd2` or `bsd3`

**Comment by domenkozar** [comment]
ping

**Comment by matthiasbeyer** [comment]
Pong.

I'm alive and I will continue to work on this asap I have time. I have exams until 19th February, but maybe I'll get this done in between.

**Comment by globin** [comment]
Merged with an update to 1.3 adding python3 support and correct license information in ae5ef2b00955a8b8cbcc93a6682a8275c4d5e52c

**Comment by matthiasbeyer** [comment]
@globin Whoa, thanks!

**Pull request closed without merging**